logic/src/commonMain/kotlin/com/wire/kalium/logic/data/session/SessionRepository.kt

```diff
@@ -23,24 +19,33 @@ class InMemorySessionRepository : SessionRepository {
         sessions[autSession.userId] = autSession
     }
 
-    override suspend fun getSessions(): Flow<Either<CoreFailure, List<AuthSession>>> = flow { emit(Either.Right(sessions.values.toList())) }
+    override suspend fun getSessions(): Either<CoreFailure, List<AuthSession>> = Either.Right(sessions.values.toList())
+    override suspend fun doesSessionExist(userId: String): Either<CoreFailure, Boolean> {
+        TODO("Not yet implemented")
+    }
 
 }
 
-class SessionRepositoryImpl(
-    private val sessionLocalDataSource: SessionLocalDataSource,
-    private val sessionMapper: SessionMapper
+class SessionDataSource(
```

Comment:
Is there any special interest on avoiding to call this class `SessionRepositoryImpl` as other classes do?

Reply:
not really, just to unify the code base, i saw other devs do it so i followed